src/Controller/UserController.php

```diff
@@ -513,4 +527,87 @@ public function getUserList(
             true
         );
     }
+
+    /**
+     * Remove User avatar.
+     * @Rest\Post("/{id}/remove_avatar")
+     * * @SWG\Post(
+     *     tags={"User"},
+     *     summary="Remove User avatar.",
+     *     description="Remove User avatar.",
+     *     operationId="removeAvatar",
+     *     produces={"application/json"},
+     *     @SWG\Parameter(
+     *     description="ID of User to edit",
+     *     in="path",
+     *     name="id",
+     *     required=true,
+     *     type="integer",
+     * ),
+     * )
+     * @SWG\Response(
+     *     response=401,
+     *     description="Unauthorized.",
+     *     @SWG\Schema(
+     *     @SWG\Property(property="code", type="integer", example=401),
+     *     @SWG\Property(property="message", type="string", example="JWT Token not found"),
+     *     )
+     * )
+     * @SWG\Response(
+     *     response="200",
+     *     description="Successfull operation!",
+     *     @SWG\Schema(
+     *     @SWG\Property(property="message", type="string", example="Avatar successfully deleted!"),
+     *     )
+     * )
+     * @SWG\Response(
+     *     response="403",
+     *     description="Forbidden",
+     *     @SWG\Schema(
+     *     @SWG\Property(property="code", type="integer", example=403),
+     *     @SWG\Property(property="message", type="string", example="Access denied!"),
+     *     )
+     * )
+     * @SWG\Response(
+     *     response=404,
+     *     description="Not found",
+     *     @SWG\Schema(
+     *     @SWG\Property(property="code", type="integer", example=404),
+     *     @SWG\Property(property="message", type="string", example="Not found!"),
+     *     )
+     * )
+     * @SWG\Response(
+     *     response=412,
+     *     description="Not found",
+     *     @SWG\Schema(
+     *     @SWG\Property(property="code", type="integer", example=412),
+     *     @SWG\Property(property="message", type="string", example="You dont have an avatar to delete!"),
+     *     )
+     * )
+     * @param User $user
+     * @param ImageRepository $imageRepository
+     * @return JsonResponse
+     * @throws ORMException
+     * @throws OptimisticLockException
+     */
+    public function removeAvatar(User $user, ImageRepository $imageRepository): JsonResponse
+    {
+        $authenticatedUser = $this->getUser();
+        if ($authenticatedUser !== $user) {
+            return new JsonResponse([
+                'code' => Response::HTTP_FORBIDDEN,
+                'message' => 'Access denied!'
+            ], Response::HTTP_FORBIDDEN);
+        }
+
+        if (!$user->getAvatar()) {
```

Comment:
No error should be returned if the avatar is not present. Just a successful message.